test/common/sku.go

```diff
@@ -0,0 +1,59 @@
+package common
```

Comment:
@psturc would you mind taking a look at this test case?

Reply:
👀